Packs/Ipstack/Integrations/Ipstack/Ipstack.py

```diff
@@ -100,5 +100,5 @@ def test_module():
         test_module()
     elif demisto.command() == 'ip':
         do_ip_command()
-except Exception, e:
+except Exception as e:
```

Comment:
This is python 2 integration, not python 3

Reply:
This syntax works in BOTH.